# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1499693897 14400
#      Mon Jul 10 09:38:17 2017 -0400
# Node ID f891afece80dfc433f8bc3bf6dcad0541abed58b
# Parent  2fc60498549564e53b02b9c00f44174819b1b94e
also substitute new mkoctfile variables in subst-cross-config-vals script

* subst-cross-config-vals.in.sh: Also substitute
OCTAVE_CONF_MKOCTFILE_OCTAVE_LINK_DEPS and
OCTAVE_CONF_MKOCTFILE_OCT_LINK_DEPS.

diff --git a/build-aux/subst-cross-config-vals.in.sh b/build-aux/subst-cross-config-vals.in.sh
--- a/build-aux/subst-cross-config-vals.in.sh
+++ b/build-aux/subst-cross-config-vals.in.sh
@@ -120,16 +120,18 @@ LIBOCTINTERP="@LIBOCTINTERP@"
 LIBS="@LIBS@"
 LLVM_CPPFLAGS="@LLVM_CPPFLAGS@"
 LLVM_LDFLAGS="@LLVM_LDFLAGS@"
 LLVM_LIBS="@LLVM_LIBS@"
 LN_S="@LN_S@"
 MAGICK_CPPFLAGS="@MAGICK_CPPFLAGS@"
 MAGICK_LDFLAGS="@MAGICK_LDFLAGS@"
 MAGICK_LIBS="@MAGICK_LIBS@"
+MKOCTFILE_OCTAVE_LINK_DEPS="@MKOCTFILE_OCTAVE_LINK_DEPS@"
+MKOCTFILE_OCT_LINK_DEPS="@MKOCTFILE_OCT_LINK_DEPS@"
 OCTAVE_LINK_DEPS="@OCTAVE_LINK_DEPS@"
 OCTAVE_LINK_OPTS="@OCTAVE_LINK_OPTS@"
 OCT_LINK_DEPS="@OCT_LINK_DEPS@"
 OCT_LINK_OPTS="@OCT_LINK_OPTS@"
 OPENGL_LIBS="@OPENGL_LIBS@"
 OSMESA_CPPFLAGS="@OSMESA_CPPFLAGS@"
 OSMESA_LDFLAGS="@OSMESA_LDFLAGS@"
 OSMESA_LIBS="@OSMESA_LIBS@"
@@ -292,16 +294,18 @@ octincludedir=`echo "@octincludedir@" | 
   -e "s|%OCTAVE_CONF_MKOCTFILE_AR%|\"${AR}\"|" \
   -e "s|%OCTAVE_CONF_MKOCTFILE_CC%|\"${CC}\"|" \
   -e "s|%OCTAVE_CONF_MKOCTFILE_CXX%|\"${CXX}\"|" \
   -e "s|%OCTAVE_CONF_MKOCTFILE_DL_LD%|\"${DL_LD}\"|" \
   -e "s|%OCTAVE_CONF_MKOCTFILE_DL_LDFLAGS%|\"${DL_LDFLAGS}\"|" \
   -e "s|%OCTAVE_CONF_MKOCTFILE_F77%|\"${F77}\"|" \
   -e "s|%OCTAVE_CONF_MKOCTFILE_LD_CXX%|\"${LD_CXX}\"|" \
   -e "s|%OCTAVE_CONF_MKOCTFILE_RANLIB%|\"${RANLIB}\"|" \
+  -e "s|%OCTAVE_CONF_MKOCTFILE_OCTAVE_LINK_DEPS%|\"${MKOCTFILE_OCTAVE_LINK_DEPS}\"|" \
+  -e "s|%OCTAVE_CONF_MKOCTFILE_OCT_LINK_DEPS%|\"${MKOCTFILE_OCT_LINK_DEPS}\"|" \
   -e "s|%OCTAVE_CONF_OCTAVE_LINK_DEPS%|\"${OCTAVE_LINK_DEPS}\"|" \
   -e "s|%OCTAVE_CONF_OCTAVE_LINK_OPTS%|\"${OCTAVE_LINK_OPTS}\"|" \
   -e "s|%OCTAVE_CONF_OCTINCLUDEDIR%|\"${octincludedir}\"|" \
   -e "s|%OCTAVE_CONF_OCTLIBDIR%|\"${octlibdir}\"|" \
   -e "s|%OCTAVE_CONF_OCT_LINK_DEPS%|\"${OCT_LINK_DEPS}\"|" \
   -e "s|%OCTAVE_CONF_OCT_LINK_OPTS%|\"${OCT_LINK_OPTS}\"|" \
   -e "s|%OCTAVE_CONF_OPENGL_LIBS%|\"${OPENGL_LIBS}\"|" \
   -e "s|%OCTAVE_CONF_OSMESA_CPPFLAGS%|\"${OSMESA_CPPFLAGS}\"|" \
